tests/e2e/vsan_test.go

```diff
@@ -101,3 +103,41 @@ func (s *VsanTestSuite) TestVSANPolicy(c *C) {
 
 	misc.LogTestStart(volCreateTest, "TestVSANPolicy")
 }
+
+// Test step:
+// 1. create a vsan policy
+// 2. run "vmdkops_admin policy ls", check the "Active" column of the output to make sure it
+// is shown as "Unused"
+// 3. create a volume on vsanDatastore with the vsan policy we created
+// 4. run "docker volume inspect" on the volume to verify the output "vsan-policy-name" field
+// 5. run "vmdkops_admin policy ls", check the "Active" column of the output to make sure it
+// is shown as "In use by 1 volumes"
+// 6. run "vmdkops_admin policy rm" to remove the policy, which should fail since the volume is still
+// use the vsan policy
+func (s *VsanTestSuite) TestDeleteVsanPolicyAlreadyInUse(c *C) {
+	misc.LogTestStart("VsanTestSuite", "TestDeleteVsanPolicyAlreadyInUse")
```

Comment:
Nit: In the existing tests we are following a convention to use the file name, i.e. "vsan_test" instead of "VsanTestSuite". Both are equally fine, but let's just keep all tests consistent.

It looks like the constant at line 41 is not being used. Let's change it to:
    testName = "vsan_test"
And use it in these tests.